powerlibs/django/restless/contrib/endpoints/base.py

```diff
@@ -71,6 +71,40 @@ def get_query_set(self, request, *args, **kwargs):
 
 
 class SoftDeletableDetailEndpointMixin:
+    def serialize(self, data):
+        data = super().serialize(data)
+        if 'deleted' in data:
+            del data['deleted']
```

Comment:
Nope.
There is already a specific way of filtering fields.

See:

https://github.com/DroneMapp/powerlibs-django-restless/blob/master/powerlibs/django/restless/modelviews.py#L69

https://github.com/DroneMapp/powerlibs-django-restless/blob/master/powerlibs/django/restless/modelviews.py#L6

https://github.com/DroneMapp/powerlibs-django-restless/blob/master/powerlibs/django/restless/models.py#L77